### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Saverio976**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,28 +2,56 @@ cmake_minimum_required(VERSION 3.3)
...
     Arcade
     VERSION 1.0
-    DESCRIPTION "A simple game engine"
+    DESCRIPTION "A simple Arcade"
     LANGUAGES CXX
 )
 
+##############################################################################
+# arcade binary
+set(CMAKE_BUILD_TYPE Debug)
```
Comment (romainpanno) [inline CMakeLists.txt:11]: debug

File: src/Arcade/Core/MainMenuScene.hpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+namespace Arcade {
+    namespace Core {
+        class MainMenuScene : public Arcade::Game::AScene {
+            public:
+                MainMenuScene(
+                    std::unique_ptr<Arcade::ECS::IEntityManager> enitityManager,
+                    const std::vector<std::pair<std::string, std::string>> &gameLibs,
+                    const std::vector<std::pair<std::string, std::string>> &graphicLibs
+                );
+                // ~MainMenuScene() = default;
```
Comment (romainpanno) [inline src/Arcade/Core/MainMenuScene.hpp:22]: dtor ?

File: src/Arcade/ECS/Entity.cpp [hunk 3]
```diff
@@ -7,7 +7,9 @@
 
 #include <algorithm>
 #include <stdexcept>
+#include <iostream>
```
Comment (romainpanno) [inline src/Arcade/ECS/Entity.cpp:10]: remove include

File: src/Arcade/LibHandler.hpp [hunk 4]
```diff
@@ -7,6 +7,7 @@
 
 #pragma once
 
+#include <cstdio>
 #include <iostream>
```
Comment (romainpanno) [inline src/Arcade/LibHandler.hpp:11]: remove include

File: src/GraphicLibs/SFML/Text.cpp [hunk 5]
```diff
@@ -0,0 +1,68 @@
+/*
+** EPITECH PROJECT, 2023
+** Arcade
+** File description:
+** Sfml
+*/
+
+#include <iostream>
```
Comment (romainpanno) [inline src/GraphicLibs/SFML/Text.cpp:8]: remove useless include

File: src/GraphicLibs/ncurses/CMakeLists.txt [hunk 6]
```diff
@@ -1,12 +1,5 @@
 cmake_minimum_required(VERSION 3.3)
```
Comment (romainpanno) [inline src/GraphicLibs/ncurses/CMakeLists.txt:1]: cmake version

**Review by romainpanno — CHANGES_REQUESTED** [review]

**Review by KitetsuK — APPROVED** [review]

File: src/Arcade/Core/MainMenuScene.hpp [hunk 7]
```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2023
+** Arcade
+** File description:
+** MainMenuScene
+*/
+
+#pragma once
+
+#include <iostream>
```
Comment (romainpanno) [inline src/Arcade/Core/MainMenuScene.hpp:10]: remove include

**Review by romainpanno — CHANGES_REQUESTED** [review]

**Comment by Saverio976** [comment]
@romainpanno

**Pull request merged**